device-types/HPE/Aruba-EC-XS-FIPS.yaml

```diff
@@ -1,20 +1,21 @@
 ---
 manufacturer: HPE
-model: Aruba ec-xs-fips
+model: Aruba EC-XS-FIPS
 slug: hpe-aruba-ec-xs-fips
 part_number: '201447'
 u_height: 1
 is_full_depth: false
 airflow: right-to-left
+weight: 3.0
```

Comment:
Looks like this file is missing the image properties
`front_image: true`
`rear_image: true`